client/service/src/builder.rs

```diff
@@ -798,7 +804,7 @@ where
 				config.network.default_peers_set.in_peers as usize +
 					config.network.default_peers_set.out_peers as usize,
 			);
-			spawn_handle.spawn("block_request_handler", handler.run());
+			spawn_handle.spawn("block_request_handler", "", handler.run());
```

Comment:
not introduced in this PR, but it would be nice to have consistent casing scheme for task names (kebab-case or snake_case, but not a mix of both)

Reply:
yes, that makes sense.